tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/transforms/legalize_to_linalg.cc

```diff
@@ -1679,6 +1679,44 @@ struct PadOpConversion : public OpConversionPattern<mhlo::PadOp> {
   }
 };
 
+/// Extract element values from `attr` and store them in `arrayValues`.
+static void getValuesFromIntAttribute(DenseIntElementsAttr attr,
+                                      SmallVector<int64_t, 4>& arrayValues) {
+  for (auto val : attr.getValues<APInt>())
```

Comment:
You should be able to do something like:

```suggestion
  for (auto val : attr.getValues<int64_t>())
```

Reply:
I think this is a useful helper function, which may have future uses. It no way depends on the lowering which I'm implementing.